### DISCUSSION TITLE
Welcome to SKILSTAK Discussions!

### DISCUSSION THREAD

**Discussion by rwxrob** [rwxrob/skilstak#25] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Afsinoz** [comment]
Hey everyone! I am Afsin, I got my master's degree in applied mathematics in France last September and I am looking for a job in tech (obviously).  I saw Skilstak and videos of Rob when I was lost and did not know what to do. I am still trying to figure it out, but even the old videos helped me, so I look forward to what we will see on the next Skilstak!

**Comment by capncoup** [comment]
Hi All! My name is Couper. I'm currently taking an introductory C++ course at my local community college. I'm fond of living a tech lifestyle and lifelong learning. I was incredibly inspired recently, having watched "The Internet's Own Boy", as I'm a person who feels strongly about politics and making a positive impact on the world. I'm motivated to take learning into my own hands, now having a taste of what my CS classes at college are like, and for this year's Beginner Boost! I'm looking forward to connecting and engaging in this community.

**Reply by lewandra** [reply]
Also felt very moved and called to action by the Aaron Swartz documentary! Also working my way through the beginner boost.
